src/plugins/guest-comment-mixin.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * This mixin is used to submit a comment on a loan that
+ * was left as a guest before they created their account.
+ * If the query parameter "guest_comment" is present and
+ * the cookie values are present, and the user is logged in,
+ * we can now leave the comment.
+ */
+import { gql } from '@apollo/client';
+import logFormatter from '@/util/logFormatter';
+
+export default {
+	mounted() {
+		const guestCommentComment = this.cookieStore.get('guestCommentComment');
+		const guestCommentInProgress = this.cookieStore.get('guestCommentInProgress');
+		const guestCommentLoanId = this.cookieStore.get('guestCommentLoanId');
+		const currentLoanId = this.$route?.params?.id;
+		const isTargetBorrowerProfile = currentLoanId.toString() === guestCommentLoanId.toString();
+		if (this.$route.query.guest_comment
```

Comment:
Could we instead just check the existence of the cookies and not worry about the query string?